src/codegen.cpp

```diff
@@ -1586,168 +1586,20 @@ void *jl_get_llvmf_decl(jl_method_instance_t *linfo, size_t world, bool getwrapp
     }
 }
 
-
-// print an llvm IR acquired from jl_get_llvmf
-// warning: this takes ownership of, and destroys, f->getParent()
-extern "C" JL_DLLEXPORT
-const jl_value_t *jl_dump_function_ir(void *f, bool strip_ir_metadata, bool dump_module)
```

Comment:
maybe confine the NFC code movements to a separate commit next time, this is kinda annoying to review with GitHub's diff view.